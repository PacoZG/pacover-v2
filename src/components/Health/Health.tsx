import React from 'react'

const Health: React.FC = () => {
  return (
    <div className="page-layout">
      <div className="sticky-header rounded-t-xl">ok</div>
    </div>
  )
}

export default Health
