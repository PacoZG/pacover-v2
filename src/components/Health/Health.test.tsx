import React from 'react'
import { render } from '@testing-library/react'
import { describe, test, expect, vi } from 'vitest'
import Health from '@/components/Health/Health'

window.scrollTo = vi.fn()

vi.mock('next-intl', () => ({
  useTranslations: () => {
    return key => {
      return key
    }
  },
}))
describe('Health component', () => {
  test('matches the snapshot', () => {
    const { container } = render(<Health />)

    expect(container).toMatchSnapshot()
  })
})
