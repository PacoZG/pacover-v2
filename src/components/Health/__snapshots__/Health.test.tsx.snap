// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Health component > matches the snapshot 1`] = `
<div>
  <div
    class="page-layout"
  >
    <div
      class="flex justify-center w-full py-8"
    >
      <div
        class="sticky-header rounded-t-xl"
      >
        ok
      </div>
    </div>
  </div>
</div>
`;
